README.md

```diff
@@ -1,20 +1,39 @@
+![ModernStorage](docs/images/favicon.png)
 # ModernStorage
 
 ModernStorage is a group of libraries that provide an abstraction layer over storage on Android to
-simplify its interactions by apps developers.
+simplify its interactions by apps developers. ModernStorage is:
+
+- **Easy to use**: ModernStorage focus on API simplicity. Rather than calling four separate methods with the Android Framework API, you should call only one with ModernStorage.
```

Comment:
```suggestion
- **Easy to use**: ModernStorage focuses on API simplicity. Rather than calling four separate methods with the Android Framework API, you only need to call one.
```